app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -2,12 +2,26 @@ module EmsRefresh::SaveInventoryContainer
   def save_ems_container_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
-    child_keys = [:container_projects, :container_quotas, :container_limits, :container_nodes,
+    graph_keys = [:container_projects, :container_quotas,
+                 ]
+    child_keys = [:container_limits, :container_nodes,
                   :container_builds, :container_build_pods, :persistent_volume_claims, :persistent_volumes,
                   :container_image_registries, :container_images, :container_replicators, :container_groups,
                   :container_services, :container_routes, :container_component_statuses, :container_templates,
                  ]
 
+    initialize_inventory_collections(ems)
+    graph_keys.each do |k|
+      send("graph_#{k}_inventory", ems, hashes[k], target)
+    end
+    ManagerRefresh::SaveInventory.save_inventory(ems, @inv_collections.values)
+
+    graph_keys.each do |k|
+      association = ems.send(k)
+      association.reset
+      store_ids_for_new_records(association, hashes[k], :ems_ref)
```

Comment:
this is kindof a performance killer that I removed fro ma new refresh, if you need this so this ties to the old refresh the ids should be already stored in @inv_collections[:container_projects].data and @inv_collections[:container_projects].data_index, ...